src/test.js

```diff
@@ -356,8 +356,8 @@ Test.prototype = {
 	},
 
 	valid: function() {
-		var include,
-			filter = config.filter && config.filter.toLowerCase(),
+		var filter = config.filter,
+			regexFilter = /^!?\/([\w\W]*)\/(i?$)/.exec( filter ),
```

Comment:
Forgot to include that in the functionality. Personally, I think it would be a nice convenience feature since writing converse regexs can get ugly. I don't see a big downside to including it, but there are some benefits.